ExampleMod/ExampleMod.cs

```diff
@@ -43,11 +40,22 @@ public class ExampleMod : Mod
 					break;
 			}
 		}
-	}
 
-	internal enum ExampleModMessageType : byte
-	{
-		ExamplePlayerSyncPlayer,
-		ExampleTeleportToStatue
+		public override void AddResearchSorting(Item item, ref ContentSamples.CreativeHelper.ItemGroup itemGroup) {
+		//Lets you add your own sorting methods in Journey mode's Duplication menu, or change the sorting methods of existing items.
+		//This is useful for items with custom damage classes, as well as specific themed items.
+			if (item.modItem?.Mod == this) { 
+				itemGroup = (ContentSamples.CreativeHelper.ItemGroup)1337; //This number is where the item sort is in relation to any other sorts added by vanilla or mods. To see the vanilla sorting numbers, refer to (insert wiki page here).
+			};
+			if (item.type == ModContent.ItemType<Content.Items.Placeable.ExampleTorch>()) {
+				itemGroup = ContentSamples.CreativeHelper.ItemGroup.Torches; //Vanilla usually matches sorting methods with the right type of item, but sometimes, like with torches, it doesn't. Make sure to set whichever items manually if need be. 
+			}
+		}
```

Comment:
Why is this hook in Mod, and not GlobalItem?

Reply:
Additionally, its name is better off being 'ModifyResearchSorting'. And there should be a ModItem version.

Reply:
This hook in general needs more development and discussion. Free-for-all id based sorting seems unnecessarily chaotic and error prone.

Reply:
This is practically the way vanilla did it, and I think it didn't matter as much for such a small thing as this.

Reply:
I agree that putting it in GlobalItem would make sense, but you barely use it for items that aren't in your mod (which is what most people use GlobalItem for). Its more of an organizational thing, like how AddRecipe is in Mod as well. I guess I could add a ModItem and GlobalItem version like AddRecipe though.